external-crates/move/crates/move-analyzer/src/bin/move-analyzer.rs

```diff
@@ -125,7 +124,8 @@ fn main() {
                 .unwrap_or(false);
         }
 
-        symbolicator_runner = symbols::SymbolicatorRunner::new(symbols.clone(), diag_sender, lint);
+        symbolicator_runner =
+            symbols::SymbolicatorRunner::new(ide_files.clone(), symbols.clone(), diag_sender, lint);
```

Comment:
Is this clone safe for in-memory VFS?

Reply:
Same answer as [here](https://github.com/MystenLabs/sui/pull/16330#discussion_r1503090998), but it's totally possible that I misunderstand Rust's behavior in this case.